restcomm/restcomm.testsuite/src/test/resources/log4j.xml.cloud

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
i guess we need to find a way to configure how this is used in JenkinsCI env...

Reply:
Agree @jaimecasero . I did this to quickly enable logs in the testsuite since they are essential when working locally with the testsuite